crates/runner-nordic/src/main.rs

```diff
@@ -77,6 +86,10 @@ struct State {
     timers: Timers,
     ccm: Ccm,
     leds: [Pin<Output<PushPull>>; <led::Impl as Support<usize>>::SUPPORT],
+    ble_radio: BleRadio,
+    ble_scanner: BeaconScanner<BleAdvScanCallback, AllowAll>,
+    ble_timer: BleTimer<TIMER0>,
+    ble_packet_queue: VecDeque<BlePacket>,
```

Comment:
Could all those fields (and most of the rest of the radio code in this file) be moved to a single struct defined in the `radio` (or ultimately `radio::ble`) module? Ideally the main should only use high-level functions for setup.